src/dsp_tools/excel2xml.py

```diff
@@ -1684,6 +1685,25 @@ def _name_label_mapper_iterator(json_subset: list[dict[str, Any]], language_labe
             # the actual values of the name and the label
 
 
+def _unescape_html_tags(xml_string: str) -> str:
```

Comment:
It is still an open question whether it should be allowed to have > and < in text properties of an XML file. By default, they are replaced by the escape sequences &gt; and &lt; by lxml. Until now, I replaced the escape sequences by the original tags in write_xml(), but this can lead to an error some lines further down in the same method, in the validation: If a text property contains for example: My text … <Fantasy> my text… , this is invalid (this actually happened to me).

The problem is non-trivial: In most cases, the tags should be escaped to prevent validation errors. But in XML-standoff formatted texts, the tags must be unescaped.

The only solution that I found is the present one. I took the list of tags from https://github.com/dasch-swiss/salsah-export/blob/master/salsah2xml.py#L74, and hope it's exhaustive.

Reply:
regarding these replacements: I think the way it should work is
- you have an input string which may or may not be XML, depending on if its rich or simple text
- in case of simple text, `<>` should be escaped because they don't bear special meaning in a plain text string, as is done by lxml
- in case of richtext, we expect it to be an XML string, so `<>` bear special meaning and must not be escaped. In this case, the string should be parsed to an XML DOM by lxml (in which case it wouldn't escape characters), and be added as child node to the `<text>` element.

in that case, the whole un-escaping should not be needed anymore